uiServices/test/testSuite.feature

```diff
@@ -0,0 +1,143 @@
+        ]
+      }
+    """
+    Then the response code should be 201
+    And the response body should be empty
+
+  Scenario: Unsuccessful creation of test Suite (file already exists )
+    Given the automated performance ui server is available
+    And the header "testSuitePathDir" is "/uiServices/test/"
+    When I send "POST" request to "/test-suites" with a body:
+    """
+      {
+       "name": "GodogTestSuite",
+       "testStrategy": "SuiteBased",
+       "testCases":[
+         {
+         "name":"file1.xml",
+          "preThinkTime": 1000,
+          "postThinkTime": 2000,
+          "execWeight": "Infrequent"
+         },
+         {
+          "name":"file2.xml",
+          "preThinkTime": 1,
+          "postThinkTime": 10,
+          "execWeight": "Sparse"
+         }
+        ]
+      }
+    """
+    Then the response code should be 400
+
+
+  Scenario: Unsuccessful creation of test Suite ( Missing Required Fields )
+    Given the automated performance ui server is available
+    And the header "testSuitePathDir" is "/uiServices/test/"
+    When I send "POST" request to "/test-suites" with a body:
+    """
+      {
+       "testCases": [
+        {
+        "name":"file1.xml",
+        "preThinkTime": 1000,
+        "postThinkTime": 2000,
+        "execWeight": "Infrequent"
+        }
+      ]
+    }
+    """
+    Then the response code should be 400
+
+
+  Scenario: Unsuccessful creation of test Suite ( No header defined )
+    Given the automated performance ui server is available
+    And the header "testSuitePathDir" is ""
+    When I send "POST" request to "/test-suites" with a body:
+    """
+      {
+       "name": "GodogTestSuite",
+       "testStrategy": "SuiteBased",
+       "testCases":[
+         {
+         "name":"file1.xml",
+          "preThinkTime": 1000,
+          "postThinkTime": 2000,
+          "execWeight": "Infrequent"
+         }
+        ]
+      }
+    """
+    Then the response code should be 400
+
+
+  Scenario: Unsuccessful creation of test Suite ( Incorrect URL )
+    Given the automated performance ui server is available
+    And the header "testSuitePathDir" is "/uiServices/test/GodogTestSuite.xml"
+    When I send "POST" request to "/xxxx" with a body:
+     """
+      {
+       "name": "GodogTestSuite",
+       "testStrategy": "SuiteBased",
+       "testCases":[
+         {
+         "name":"file1.xml",
+          "preThinkTime": 1000,
+          "postThinkTime": 2000,
+          "execWeight": "Infrequent"
+         }
+        ]
+      }
+    """
+    Then the response code should be 405
+
+  Scenario: Unsuccessful creation of test Suite(No Name defined)
+    Given the automated performance ui server is available
+    And the header "testSuitePathDir" is "/uiServices/test/GodogTestSuite.xml"
+    When I send "POST" request to "/test-suites" with a body:
+     """
+      {
+       "name": "",
+       "testStrategy": "SuiteBased",
+       "testCases":[
+         {
+         "name":"file1.xml",
```

Comment:
we should go with config approach of appending .xml separately to the name, right? name and file name can be independent.